flumine/config.py

```diff
@@ -5,7 +5,7 @@
 
 instance_id = None  # instance id (e.g. AWS ec2 instanceId)
 
-hostname = socket.gethostname()[
+customer_strategy_ref = socket.gethostname()[
```

Comment:
Can you add an alias below 

```python
hostname = customer_strategy_ref
```

We can then add a warning before removing it in the history as I know a few have used/changed this.